pkg/assembler/nodes.go

```diff
@@ -29,11 +34,12 @@ func (an ArtifactNode) Properties() map[string]interface{} {
 	properties := make(map[string]interface{})
 	properties["name"] = an.Name
 	properties["digest"] = an.Digest
+	properties["source"] = an.Metadata.Source
```

Comment:
i think we can make a generic function for this? like populate metadata ? And do the same for PropertyNames() and others as well? instead of manually populating them?

Reply:
made it a map[string]interface{} so can add properties in the future. Added constants for the property names.